src/verification/apple.rs

```diff
@@ -37,6 +37,7 @@ fn system_time_to_cfdate(time: SystemTime) -> Result<CFDate, TlsError> {
         .map(|epoch| CFDate::new(epoch.as_secs() as f64))
 }
 
+/// A TLS certificate verifier that uses the system's root store and WebPKI.
```

Comment:
I don't believe this is correct, was this a copy/paste error from the `other` verifier? We use both the system's root store and Apple's TLS verification routines that come from `Security.framework`. This one should be a closer match to the comment for Windows.

Reply:
Copy-pasta err :spaghetti: :x: Thanks for catching.

Reply:
Fixed up.